components/CommunitySingle/CommunitySingle.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Card, Box, Button, DefaultCard, CardGrid } from 'ui-kit';
+import { themeGet } from '@styled-system/theme-get';
+import { SEO, CustomLink } from 'components';
+
+function CommunitySingle(props = {}) {
+  return (
+    <>
+      <SEO title={props.data?.title} />
+      <Box
+        height={{ _: '298px', md: '596px' }}
+        mb="l"
+        textAlign="center"
+        flexDirection="column"
+        backgroundImage={`linear-gradient(rgba(0, 0, 0, 0.33), rgba(71, 71, 71, 0.85)),url(${props.data?.coverImage?.sources[0]?.uri})`}
+        backgroundSize="cover"
+        color="white"
+        justifyContent="center"
+        alignItems="center"
+        display="flex"
+        borderRadius="base"
+        mx="xxl"
```

Comment:
Not a huge deal to hold anything up, but this is a lot of inline style props. We might want to consider pulling this into its own Styled Component…